package.json

```diff
@@ -20,7 +20,9 @@
     "Other"
   ],
   "preview": true,
-  "activationEvents": [],
+  "activationEvents": [
+    "onStartupFinished"
```

Comment:
We're actually gonna avoid eager loading for now.
Extensions should activate when actually used